types/ember__object/test/octane.ts

```diff
@@ -47,6 +53,17 @@ class Foo extends EmberObject {
         return `${this.firstName} ${this.lastName}`;
     }
 
+    @computed("firstName", "lastName") // $ExpectError
```

Comment:
For my own knowledge: does this test work b/c we're returning a `MethodDecorator` here? I.e. `MethodDecorator`s work on getters but not on regular properties?

Reply:
That's right — `MethodDecorator`s receive the descriptor as an argument, while `PropertyDecorator`s (at least as far as TS's implementation is concerned) only receive the target and the key.